styles.css

```diff
@@ -29,34 +29,34 @@ hr{
 }
 
 .container-head {
-border: 1px solid none;
-display: flex;
-flex-flow: row wrap;
-justify-content: flex-start;
-align-items: center;
-gap: 1px;
+  border: 1px solid none;
```

Comment:
i believe you don' t need to write `none`, if you want to use the same color your element has, it should be default

Reply:
okay, I will check this later, thanks for the advise :-)